pages/common/odps.md

```diff
@@ -0,0 +1,39 @@
+# odps
+
+> Aliyun ODPS command line tool.
+
+- Start the command line:
+
+`odpscmd --config {{odps_config.ini}}`
```

Comment:
Is there a default config? Ideally we'd start with that, the simplest example, but I see this tool is quite versatile, so we can use --config here, but we should mention it in the command description -- e.g. "Start the command line with a custom configuration file".